_src/src/utils.js

```diff
@@ -0,0 +1,9 @@
+export default {
+    getLanguage() {
+        if (navigator.language) {
+            return navigator.language;
```

Comment:
`window.navigator.language` may give different results depending on the browser (for the same language). For me, Chrome returns `pl`, while Edge `pl-PL`, need to take this into account.

Reply:
I was add some code to match pl-PL as pl, but I think is little ugly..